### ISSUE TITLE
Could you please give an example of Image Classification Training using TN?

### ISSUE THREAD

**Issue by jackiechen0708** [google/TensorNetwork#44]
" The construction of these special MPS tensor networks amounts to the training of the model, the details of which can be found in the work of Stoudenmire and Schwab [35], as well as in our TensorNetwork implementation [47]."

I have read the paper and found that the tensor network can be used in Image Classification.But I do not know the detail. And the "TensorNetwork on TensorFlow: An Application to Machine Learning" is not available now.

So could you please give an example of Image Classification Training using TN(TT format)?

**Comment by Thenerdstation** [comment]
Yes! We have an intern who got a working example using gradient decent.

We should get it out soon. Our implementation uses the new `tf.vectorized_map` that is only available on the nightly build.

**Comment by jackiechen0708** [comment]
Wow!That's really a wonderful work!Looking forward to your release~

**Comment by Thenerdstation** [comment]
This exists now in the `examples/MPS_classifier` directory
